### ISSUE TITLE
zeroUnits false negative for relative sizes

### ISSUE THREAD

**Issue by mehcode** [rossPatton/stylint#163]
```stylus
html, body {
  height: 100%;
}
```

```
Warning: Including the unit value is preferred
File: [...]
Line: 2: height: 100%;

Stylint: 0 Errors.
Stylint: 1 Warnings.
```

---

```
$ stylint --version
Stylint version: 1.0.9
```

**Comment by rossPatton** [comment]
1.0.10 is out now, this issue should be fixed. Lemme know if you still have issues.
